coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -78,40 +79,95 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
 
         return file_diff_dict
 
-    def get_ignore_comment(self, origin, language):
+    def get_ignore_comment(self, language, spaces: str=''):
```

Comment:
please restore `origin`.  removing it is a breaking change.
Do not do that.

Add docstring params to state it is unused.

Also explain what `spaces` should be in the docstring.